cli/tox.ini

```diff
@@ -128,6 +128,7 @@ deps =
     bandit
 commands =
     bandit -r \
+        --skip B105,B404 \
```

Comment:
can you add a comment on why we disable these 2 rules globally?

Reply:
Yes. I'm adding inline comments to the bandit config file.